webapp/_lib/view/_header.tpl

```diff
@@ -2,7 +2,7 @@
 <html lang="en" prefix="og: http://ogp.me/ns#" itemscope itemtype="http://schema.org/Article">
 <head>
     <meta charset="utf-8">
-    <title>{if $controller_title}{$controller_title} | {/if}{$app_title}</title>
```

Comment:
In this deleted line, the page uses $controller_title if it is set. Why the change?